src/app/organisms/room/common.jsx

```diff
@@ -173,11 +206,19 @@ function parseTimelineChange(mEvent) {
     variant,
     content,
   });
+  const type = mEvent.getType();
   const content = mEvent.getContent();
   const prevContent = mEvent.getPrevContent();
   const sender = mEvent.getSender();
   const senderName = getUsername(sender);
   const userName = getUsername(mEvent.getStateKey());
+  
+  
+  switch (type) {
+    case 'm.room.name': return makeReturnObj('edit', tJSXMsgs.roomNameChanged(senderName, content.name));
+    case 'm.room.topic': return makeReturnObj('edit', tJSXMsgs.roomTopicChanged(senderName));
+    case 'm.room.avatar': return makeReturnObj('edit', tJSXMsgs[content.url ? 'roomAvatarChanged' : 'roomAvatarRemoved'](senderName));
+  }
```

Comment:
It's always a good practice to have deafult case.

Reply:
in this case we *want* the case to fall through if it's not handled by any of these cases, so that it can be caught by the second switch/case a little bit below